tests/puppeteer/campaigns/data/demo/products.js

```diff
@@ -1,6 +1,23 @@
 module.exports = {
   Products: {
-    demo_14: {id: '19', name: 'Customizable mug', reference: 'demo_14'},
-    demo_1: {id: '1', name: 'Hummingbird printed t-shirt', reference: 'demo_1'},
+    demo_1: {
+      id: 1, name: 'Hummingbird printed t-shirt', reference: 'demo_1', category: 'Men', status: 'Active',
```

Comment:
```suggestion
      id: 1,
      name: 'Hummingbird printed t-shirt', 
      reference: 'demo_1', 
      category: 'Men',
      status: 'Active',
```
Maybe it's more visible like this for all of them, what do you think ?